code/src/reducers/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem("state");
```

Comment:
Great that you persist your state inside the localStorage!! It is a great skill when you want to build your shopping cart for example or provide multiple themes to choose from.